libs/utils/utils.js

```diff
@@ -461,6 +461,16 @@ export async function loadBlock(block) {
         await init(block);
       } catch (err) {
         console.log(`Failed loading ${name}`, err);
+
+        const { message, errorType, module } = error;
+        if (message && errorType && module) {
+          window.lana.log(`${message}; blockPath: ${blockPath} | href: ${window.location.href} | ${err.reason || err.error || err.message || err}`, {
+            clientId: 'feds-milo',
+            sampleRate: 1,
+            tags: `errorType=${errorType},module=${module}`,
+          });
+        }
```

Comment:
We should find another way to expose errors in the blocks themselves. utils could dispatch an error or something of the sorts, but handling should be done in the block.